Remove data and src symlinks

jetm opened the pull request:
It makes data and src directories, instead of symlinks.

This fixes a weird issue when the indexed data is mounted outside of the Docker container.

Related to #27.

vladak commented:
I believe `scripts/index.sh` needs to be changed as well - notably the args to -s and -d Indexer options.

vladak commented:
Actually, `scripts/start.sh` needs to be changed as well, it contains references to `/data`.

Commit:
Remove data and src symlinks

It makes data and src directories, instead of symlinks. Move data and
src under /opengrok directory.

It fixes a weird issue when the indexed data is mounted outside of the
Docker container.

vladak commented:
Could you rebase please ?

jetm commented:
@vladak Rebased

vladak commented:
Thanks !
